src/engine/ruleset/integrations/docker-listener/decoders/docker-listener.yml

```diff
@@ -0,0 +1,202 @@
+name: decoder/docker-listener/0
+
+metadata:
+  title: Wazuh docker listener integration
+  description: Process events that comes from the Docker listener wodle
+  module: Docker-listener
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.4 and docker 20.10.12
+  versions:
+   - "4.4"
+   - "20.10.12" #TODO: check backward compatibility
+  author:
+    email: info@wazuh.com
+    name: Wazuh, Inc.
+    date: 2023-06-01
+  references:
+    - https://documentation.wazuh.com/current/container-security/docker-monitor/index.html
+    - https://docs.docker.com/engine/reference/commandline/events/
+
+sources:
+  - decoder/integrations/0
+
+check:
+  - wazuh.origin: Wazuh-Docker
+  - event.original: +starts_with/{
+
+parse:
+  logpar:
+    - event.original: <~json/json>
+
+definitions:
+  MAP_ContainerStatusToEventType:
+    "attach": "access,connection" #TODO: check with double values
+    "commit": "creation"
+    "copy": "change"
+    "create": "creation"
+    "destroy": "deletion"
+    "detach": "connection"
+    "die": "end"
+    "exec_create": "creation,start"
+    "exec_detach": "change,start"
+    "exec_die": "creation,end"
+    "exec_start": "creation,start"
+    "export": "change"
+    "health_status": "info"
+    "kill": "end"
+    "oom": "error"
+    "pause": "change"
+    "rename": "change"
+    "resize": "change"
+    "restart": "start"
+    "start": "start"
+    "stop": "end"
+    "top": "info"
+    "unpause": "start"
+    "update": "change"
+
+  MAP_ImageActionToEventType: #TODO: Double check
+    "delete": "deletion"
+    "import": "creation"
+    "load": "start"
+    "pull": "start"
+    "push": "creation"
+    "save": "change"
+    "tag": "indicator"
+    "untag": "indicator"
+
+  MAP_PluginActionToEventType:
+    "enable": "start"
+    "disable": "end"
+    "install": "creation"
+    "remove": "deletion"
+
+  MAP_VolumeActionToEventType:
+    "create": "creation"
+    "destroy": "deletion"
+    "mount": "start"
+    "unmount": "end"
+
+  MAP_NetworkActionToEventType:
+    "create": "creation"
+    "connect": "connection,start"
+    "destroy": "deletion,end"
+    "disconnect": "connection,end"
+    "remove": "deletion"
+
+  MAP_BaseActionToEventType:
+    "create": "creation,deletion"
+    "remove": "deletion"
+    "update": "change"
+
+normalize:
+
+ - map:
+    - wazuh.decoders: +array_append/docker-listener
+
+ - check: +exists/~json.docker
+   map:
+    - event.module: docker-listener
+    - event.kind: event #alert or metric
+    - event.outcome: unknown
+    - container.id: $~json.docker.id
+    - event.created: +date_from_epoch/$~json.docker.time
+
+  # Containers
+ - check: +string_equal/~json.docker.Type/container
```

Comment:
I think that check list is more readable, the same for the other checks

Reply:
Changed!